.drone.yml

```diff
@@ -8,37 +8,391 @@ platform:
   version: 1809
 
 steps:
-  - name: build
-    pull: default
-    image: maiwj/dapper:windows-1809
-    commands:
-      - dapper.exe -f Dockerfile.dapper ci
+- name: build
```

Comment:
we might want to talk about what youre trying to accomplish but i dont think your images outside of 1809 will have binaries if you dont build them on each build host

Reply:
Ah good catch! I didn't realize I omitted the build step on each host. Adding to the next commit